src/Tutor.Infrastructure/Database/EventStore/PostgreSQLEventStore/EventContext.cs

```diff
@@ -5,19 +5,18 @@ namespace Tutor.Infrastructure.Database.EventStore.PostgreSqlEventStore
 {
     public class EventContext : DbContext
     {
-        internal DbSet<EventWrapper> Events { get; set; }
+        internal DbSet<StoredDomainEvent> Events { get; set; }
 
         public EventContext(DbContextOptions<EventContext> options) : base(options)
         {
         }
 
         protected override void OnModelCreating(ModelBuilder modelBuilder)
         {
-            modelBuilder.Entity<EventWrapper>().Property(e => e.DomainEvent).HasColumnType("jsonb");
-            modelBuilder.Entity<EventWrapper>().Property(e => e.DomainEvent).HasConversion(
+            modelBuilder.Entity<StoredDomainEvent>().Property(e => e.DomainEvent).HasColumnType("jsonb");
+            modelBuilder.Entity<StoredDomainEvent>().Property(e => e.DomainEvent).HasConversion(
             v => EventSerializer.Serialize(v),
             v => EventSerializer.Deserialize(v));
-            modelBuilder.Entity<EventWrapper>().HasIndex(e => e.Timestamp);
```

Comment:
For now I removed the timestamp from <code>EventWrapper</code>/<code>StoredDomainEvent</code> to avoid duplicating any data from <code>DomainEvent</code>. However, that does mean that the timestamp is currently only stored in the jsonb column, which causes issues with indexing. I haven't found a way to create an index over a jsonb column with EF Core (other than directly executing SQL), and I'm also not sure how significant the difference in performance between regular indexes and jsonb indexes is.